components/mpas-ocean/src/shared/mpas_ocn_surface_land_ice_fluxes.F

```diff
@@ -677,14 +678,24 @@ subroutine ocn_surface_land_ice_fluxes_init(err)!{{{
 
       select case (trim(config_land_ice_flux_mode))
       case ('standalone','Standalone','STANDALONE')
-         landIceFluxesOff = .false.
-         standaloneOff    = .false.
+         landIcePressureOn = .true.
+         landIceFluxesOn = .true.
+         standaloneOn    = .true.
       case ('coupled','Coupled','COUPLED')
-         landIceFluxesOff = .false.
-         standaloneOff    = .true.
+         landIcePressureOn = .true.
+         landIceFluxesOn = .true.
+         standaloneOn    = .false.
+      case ('pressure_only','Pressure_Only','PRESSURE_ONLY')
+         landIcePressureOn = .true.
+         landIceFluxesOn = .false.
+         standaloneOn    = .false.
+      case ('off','Off','OFF')
+         landIcePressureOn = .false.
+         landIceFluxesOn = .false.
+         standaloneOn    = .false.
```

Comment:
@xylar thanks for the suggestion. I think this is much improved, we should never not use positive logic ;).  I also made the logic flags public if they are used elsewhere (specifically diagnostics). 

I thought about substituting `couplingOn` for `standAloneOff` but that doesn't make sense for these last two cases, i.e. `standAloneOff` ( = `.not.standAloneOn`) is really what is needed to run the subroutine `ocn_surface_land_ice_fluxes_build_arrays`.